### PR TITLE
Avs/2020 07 17 preview

### PR DESCRIPTION
**Pull request by gapariha**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by gapariha** [comment]
@ctaggart - could you pls do an early review and provide your inputs.

**Comment by gapariha** [comment]
Please do NOT merge this pull request. I have created this one to get early review feedback.

**Comment by gapariha** [comment]
@naman1901, @SasankBSN96 - please review PR changes.

**Comment by ctaggart** [comment]
As mentioned in email, please move to private repository at https://github.com/Azure/azure-rest-api-specs-pr and base it off its master branch.

**Comment by gapariha** [comment]
Will do, thanks.

File: specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json [hunk 1]
```diff
@@ -4186,6 +5044,391 @@
...
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DRSGroupEntity": {
```
Comment (SasankBSN96) [inline specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json:5144]: nit: can we just call it DRSGroup?

Comment (SasankBSN96) [inline specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json:5144]: nit: can we call it just DRSGroup?

File: specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json [hunk 2]
```diff
@@ -4186,6 +5044,391 @@
...
+            "modelAsString": true
+          }
+        }    
+      },
+      "required": [
+        "groups",
+        "affinityType"
+      ]    
+    },
+    "PolicyDRSVmOverride": {
```
Comment (SasankBSN96) [inline specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json:5382]: Should we name this as "PolicyDisableDRSVmOverride"

**Pull request closed without merging**